bindings/py/cpp_src/bindings/algorithms/py_TemporalMemory.cpp

```diff
@@ -47,7 +47,7 @@ Example usage:
     TODO
 )");
 
-	py::enum_<TemporalMemory::ANMode>(m, "ANMode")
+	py::enum_<TemporalMemory::ANMode>(m, "ANMode") //TODO currently htm.bindings.algorithms.ANMode, make ANMode part of algorithms.TemporalMemory
```

Comment:
nit, how would it be possible to make enum `ANMode` exposed as a part of `htm.bindings.algorithms.TemporalMemory.ANMode`, and not the current `..bindings.algorithms.ANMode` ?